### PR TITLE
KAN-53 - partial changes for review

### PR DESCRIPTION
**Pull request by ischenkx**

### CODE REVIEW

File: backend/stand/docker-compose.yml [hunk 1]
```diff
@@ -17,6 +17,22 @@ services:
       - postgres
       - redis
       - depot
+  taskdep:
```
Comment (DenShlk) [inline backend/stand/docker-compose.yml:20]: please git ignore binaries (nothing wrong with this file, i can comment only on lines of code)

File: framework/plugins/futdep/manager.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+func NewManager(deps deps.Deps, fut2dep kv.Database[future.ID, string]) *Manager {
+	return &Manager{
+		fut2dep: fut2dep,
+		deps:    deps,
+	}
+}
+
+func (manager *Manager) Make(ctx context.Context, id future.ID) (string, error) {
+	dependencyID, err := manager.fut2dep.Get(ctx, id)
```
Comment (DenShlk) [inline framework/plugins/futdep/manager.go:24]: please customize errors

File: framework/future/manager.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		return err
+	}
+	if !set {
+		return ErrAlreadyResolved
+	}
+	manager.runner.Run(ctx, Resolution{Future: future, Resource: resource})
+	return nil
+}
+
+func (manager *Manager) Load(ctx context.Context, id ID) (Resolution, error) {
```
Comment (DenShlk) [inline framework/future/manager.go:51]: may be return ErrNotResolved if can't get the resource?

**Review by DenShlk — CHANGES_REQUESTED** [review]
pr is huge and has no tests (except for stand), so i haven't checked for bugs:)

File: kernel/kernel.go [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+	"github.com/google/uuid"
+	"kantoku/kernel/platform"
+)
+
+type Kernel struct {
+	platform platform.Platform[Task]
+	plugins  []Plugin
+}
+
+func New(platform platform.Platform[Task]) *Kernel {
```
[the remark this answers is not in the document]
Comment (ischenkx) [reply kernel/kernel.go:14]: I formatted the whole project

File: backend/stand/docker-compose.yml (hunk 1 above)
Comment (ischenkx) [reply backend/stand/docker-compose.yml:20]: I removed all binaries

File: framework/future/manager.go (hunk 3 above)
Comment (ischenkx) [reply framework/future/manager.go:51]: Yup

**Review by DenShlk — APPROVED** [review]
looks better now, good work🚀

**Pull request merged**